src/main/java/com/adamk33n3r/runelite/watchdog/alerts/PlayerChatAlert.java

```diff
@@ -0,0 +1,29 @@
+package com.adamk33n3r.runelite.watchdog.alerts;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class PlayerChatAlert extends Alert implements RegexMatcher {
```

Comment:
Wonder if it makes sense to extend from `ChatAlert1`. then all you'd need to have is the constructor for the new label. Cause really, all this class is is kind of a "flag" to change how the event handler works. It's not adding new specific functionality itself.

Reply:
I think I'm following. I'll push a revision and let me know if this is what you thought.